setup.py

```diff
@@ -141,6 +141,9 @@ def render_tmpl(template, mode=None, is_yaml=False):
     "sysvinit_openrc": lambda: [
         f for f in glob("sysvinit/gentoo/*") if is_f(f)
     ],
+    "sysvinit_openrc.dep": lambda: [
+        f for f in glob("tools/cloud-init-hotplugd") if is_f(f)
+    ],
```

Comment:
No need to glob here since we are just providing one known file with this lambda
```suggestion
    "sysvinit_openrc.dep": lambda: ["tools/cloud-init-holplugd"],
```

Reply:
I've pushed this suggestion to setup.py and will land when CI clears. Thanks again @dermotbradley